packages/rx_bloc_cli/README.md

```diff
@@ -49,6 +50,10 @@ Out of the box, a Rx Bloc CLI created projects includes:
 
 ✅ [Push notifications][push_notifications_lnk] - (Coming soon) Receive push notifications and messages to your device using Firebase Cloud Messaging
 
+✅ [Http client][dio_http_client_lnk] - Dio is an Http client, which supports Interceptors, Global configuration, FormData, Request Cancellation, File downloading, Timeout etc.
+
+✅ [Interceptors][interceptors_lnk] - Interceptors are a powerful mechanism that can monitor, rewrite, and retry calls.
```

Comment:
Since there is a mention of the interceptors in the http client's description and since they are used along with that client, we can remove this line explaining what they are. 
However, you can still add the link where the interceptors are mentioned in the clients description.

Reply:
Ready.